test_scripts/API/Registration/013_SDLVersion_only_once_use_value_durin_ignition_cycle.lua

```diff
@@ -0,0 +1,50 @@
+---------------------------------------------------------------------------------------------------
+-- Regression check
+-- User story:TBD
+-- Use case:TBD
+--
+-- Requirement summary:
+-- TBD
+--
+-- Description:
+-- Check that SDL returns SDL version in RegisterAppInterface response.
+-- In case:
+-- 1) Update into .ini file param "SDLVersion".
+-- 2) SDL, HMI are started.
+-- 3) Mobile application is registered with updated "SDLVersion".
+-- SDL does:
+-- 1) Not send and return SDL version in RegisterAppInterface response.
```

Comment:
@GetmanetsIrina It's hard to understand this sentence: `SDL does Not send and return`